docs/content/en/docs/concepts/_index.md

```diff
@@ -44,6 +44,14 @@ For example, to treat any context as local by default:
 skaffold config set --global local-cluster true
 ```
 
+## Environment configuration
+
+Some environment or developer specific settings can be configured using environment variables. The options are
+
+| Variable Name | Description |
+| ------ | ----------- |
+| `KUBE_CONTEXT` | The kubectl context to use for deployment. |
```

Comment:
I'm a bit worried about setting the kubecontext from the environment. There is a lengthy discussion about exactly that env variable around `kubectl`: https://github.com/kubernetes/kubernetes/issues/27308 . This would preclude a future command line flag `--context` which would be more in line with other kubernetes tools like `kubectl`.
@kaperys Does that solve a particular problem you have, or did you just add it for good measure?

Reply:
Mostly for good measure - some earlier comments in #511 suggested this change. Supporting the `context` property in the skaffold.yml schema will solve my particular problem.